src/main/java/me/leoko/advancedban/bukkit/BukkitMethods.java

```diff
@@ -75,12 +76,15 @@ public String getFromUrlJson(String url, String key) {
             JSONParser jp = new JSONParser();
             JSONObject json = (JSONObject) jp.parse(new InputStreamReader(request.getInputStream()));
 
-            String[] keys = key.split("\\|");
```

Comment:
This code enabled the user to define a key which points to a child JsonObject.
Like result|playerdata|uuid
would point to result: { playerdata: { uuid: VALUE}}
Is there any reason for you to remove it? ^^